application/src/main/java/org/thingsboard/server/service/state/DefaultDeviceStateService.java

```diff
@@ -285,7 +284,6 @@ public void onDeviceInactivityTimeoutUpdate(TenantId tenantId, DeviceId deviceId
         log.trace("on Device Activity Timeout Update device id {} inactivityTimeout {}", deviceId, inactivityTimeout);
         DeviceStateData stateData = getOrFetchDeviceStateData(deviceId);
         stateData.getState().setInactivityTimeout(inactivityTimeout);
-        checkAndUpdateState(deviceId, stateData);
```

Comment:
I am still trying to figure out what this fix is about.

If I increase the inactivity timeout - the state has to be recalculated (same as restarting the application with a new timeout)
If I decrease the inactivity timeout - the state also has to be recalculated (same as application restart)

The only change I can imagine is not to fire `onDeviceActivity` when attribute `inactivityTimeout` was changed by TransportToDeviceActorMsg